### DISCUSSION TITLE
63US YI DOME 720P

### DISCUSSION THREAD

**Discussion by hecabb** [alienatedsec/yi-hack-v5#383] [Q&A] [answered]
Hi is there a hack for this model - 63US DOME 720P ?
YI hackv4 is only that worked but what i can see that project is dead.

**Comment by alienatedsec** [comment]
It will work. Please follow the official guide.

**Reply by hecabb** [reply]
Have You tested it or someone with this old model of Dome?
I've made step by step from guide even with different cards 8GB and 32GB and did not managed to make it work. 
The camera is boot looping and is not connecting to wifi. Tested also with 0.3.8 and it connected to wifi after installation but the website of camera ip is not working :(
Official software works and only this V4 from theCrypt0 version works.

**Comment by hecabb** [comment, marked as answer]
Ok I finally figured it out, my wifi network runs on "DUAL-BAND Smart Connect", it automatically assigns the network to either a 2.4GHz or 5GHz device. Apparently it's having some sort of problem assigning itself to the correct network and is getting into bootloop when trying to connect to the network. I'm also using the AllWinner hack on other YI cameras and haven't had this problem there. 
I made a guest network just on this frequency and now it works.
